everything/sop/pos-order-refresh-failure.md

```diff
@@ -0,0 +1,59 @@
+---
+description: >-
+This document provides steps to resolve the 'POS Order Refresh Failure' issue in Hotwax Commerce OMS.
+---
+
+# POS Order Refresh Failure in Hotwax Commerce OMS
+
+
+POS orders are downloaded as fulfilled in HotWax Commerce from Shopify. However, there are instances when the order is downloaded in OMS before it's marked fulfilled in Shopify. In such cases, retailers have to refresh the order to ensure fulfilled orders are downloaded while the unfulfilled ones get canceled.
+
+However, sometimes the orders are not refreshed due to missing shipping addresses. The error message received when hitting the “Refresh Order” button is "Could not complete the createOrderContactMech process: The following required parameter is missing: [createOrderContactMech.contactMechId]".
+
+In such cases, such orders need to be manually removed, and the updated order needs to be imported again.
```

Comment:
i think this is the same as I had written, need to ChatGPT it once

Reply:
Okay sir